RELEASE_NOTES.md

```diff
@@ -1,5 +1,10 @@
 # Release Notes
 
+## 4.8.2
+### Bugfixes
+- Fixed a bug in the MacOS pkginstaller where Podman machine was using a different QEMU binary than the one installed using the installer, if it existed on the system ([#20808](https://github.com/containers/podman/issues/20808)).
+- Fixed a bug on Windows (WSL) with the first-time install of user-mode networking when using the init command, as opposed to set ([#20921](https://github.com/containers/podman/issues/20921)).
+
```

Comment:
Should we mention that it also includes the fix for https://github.com/containers/podman/issues/20432?

Reply:
sure I can do that

Reply:
Done. PTAL